test/tests/functional/pbs_fairshare.py

```diff
@@ -157,10 +157,14 @@ def test_fairshare_formula3(self):
         self.server.manager(MGR_CMD_SET, SERVER, {'job_sort_formula': formula})
         J1 = Job(TEST_USER2)
         jid1 = self.server.submit(J1)
+        # Adding sleep for different job submission timestamp
+        time.sleep(1)
```

Comment:
I don't see the need to have different timestamps for the different jobs.  The test just needs to have 4 of them submitted before one cycle.  The test is matching the scheduler log, not anything to do with the job times.

Reply:
you are right. I will remove it.

Reply:
There were other sleeps added in other tests.  Are they needed?

Reply:
I see what you are saying. I saw a couple of tests failing due to political order mismatch. Unfortunately I do not have the sched_logs and I am not able to reproduce this now. I will remove the sleep from all the tests.

Reply:
If you want to enforce a political ordering, set the job_sort_formula to 'job_priority' and submit the jobs with the priority you want.  The jobs with the highest priority will be considered first.